TerminalDocs/customize-settings/profile-settings.md

```diff
@@ -0,0 +1,423 @@
+
+<br />
+
+___
+
+## Executable settings
+
+### Command line
+
+The executable used in the profile.
+
+**Property name:** `commandline`
+
+**Necessity:** Optional
+
+**Accepts:** Executable file name as a string
+
+### Source
+
+Stores the name of the profile generator that originated this profile. _There are no discoverable values for this field._
+
+**Property name:** `source`
+
+**Necessity:** Optional
+
+**Accepts:** String
+
+### Starting directory
+
+The directory the shell starts in when it is loaded.
+
+**Property name:** `startingDirectory`
+
+**Necessity:** Optional
+
+**Accepts:** Folder location as a string
+
+**Default value:** `"%USERPROFILE%"`
+
+<br />
+
+___
+
+## Dropdown settings
+
+### Name
+
+This is the name of the profile that will be displayed in the dropdown menu. This value is also used as the "title" to pass to the shell on startup. Some shells (like `bash`) may choose to ignore this initial value, while others (`cmd`, `powershell`) may use this value over the lifetime of the application. This "title" behavior can be overridden by using `tabTitle`.
+
+**Property name:** `name`
+
+**Necessity:** Required
+
+**Accepts:** String
+
+### Icon
+
+Set the icon that displays within the tab and the dropdown menu.
+
+**Property name:** `icon`
+
+**Necessity:** Optional
+
+**Accepts:** File location as a string
+
+### Hide a profile from the dropdown
+
+If `"hidden"` is set to `true`, the profile will not appear in the list of profiles. This can be used to hide default profiles and dynamically generated profiles, while leaving them in your settings file.
+
+**Property name:** `hidden`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+**Default value:** `false`
+
+<br />
+
+___
+
+## Tab title settings
+
+### Custom tab title
+
+If set, this will replace the `name` as the title to pass to the shell on startup. Some shells (like `bash`) may choose to ignore this initial value, while others (`cmd`, `powershell`) may use this value over the lifetime of the application. If you'd like to learn how to have the shell set your title, visit the [tab title tutorial](./../tutorials/tab-title.md).
+
+**Property name:** `tabTitle`
+
+**Necessity:** Optional
+
+**Accepts:** String
+
+### Suppress title changes from the shell
+
+When set to `true`, `tabTitle` overrides the default title of the tab and any title change messages from the application will be suppressed. If `tabTitle` isn't set, `name` will be used instead. When set to `false`, `tabTitle` behaves as normal.
+
+**Property name:** `suppressApplicationTitle`
+
+**Necessity:** Optional
+
+**Accepts:** `true`, `false`
+
+<br />
+
+___
+
+## Text settings
+
+### Font face
+
+Name of the font face used in the profile. The Terminal will try to fallback to Consolas if this can't be found or is invalid. For other variants of the default font, Cascadia Code, visit the [Cascadia Code GitHub repository](https://github.com/microsoft/cascadia-code).
```

Comment:
Should we also describe that we will follow a fallback strategy for glyphs missing from this font when rendering? And that they'll be scaled to try to fit inline with the chosen font?